test/test_utils.py

```diff
@@ -168,6 +169,13 @@ def test_draw_invalid_boxes():
         utils.draw_bounding_boxes(img_correct, boxes, colors=colors_wrong)
 
 
+def test_draw_boxes_warning():
+    img = torch.full((3, 100, 100), 255, dtype=torch.uint8)
+
+    with pytest.warns(UserWarning, match=re.escape("Argument 'font_size' will be unused as 'font' is not set.")):
```

Comment:
I don't think we need `re.escape`?

Reply:
Resolving this after offline discussion. We technically don't need it here, since the only special regex character here is the `.` at the end. Since it matches anything, the test would also pass without escaping the string.

IMO, it is still useful here to communicate the intent. Using it makes it clear that we want an exact match. Whether or not we should want this in the first place is a different discussion.